### PR TITLE
Added FAQ

### PR DESCRIPTION
**Pull request by ericsjogren**
The idea was that the Goalkeeper will add questions received over email in this document, so we can build a base of FAQs to relieve some of the effort required in the future

### CODE REVIEW

**Comment by ericsjogren** [comment]
cant find iZettle/commerce-backend as reviewer

File: faq.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+> No, its completely free to use our API and the only limitation is that some resources and
+services might have rate-limiting enabled to prevent overload.
+
+___
+##### Question:
+> Do you provide any test environment accounts?
+
+##### Answer:
+> Unfortunately we do not at this time, there are however test accounts in production that you can use.
```
Comment (psvanstrom) [inline faq.adoc]: No, the accounts listed in the authz documentation is for the sandbox environment we used initially when building authorization. That environment still work for testing the OAuth flow and some parts of purchase/finance but hasn't been updated in a long time and should probably be decommissioned.

There are no test accounts in production available to external integrators. They will need real accounts for the time being.

File: faq.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+> This is something our framework sends when it has received the request headers and
+indicates that the client should proceed to send the request body. What you can do is investigate
+ the reason why your code does this, or you could traverse the headers until
+you find a header with the intended response code, examples [Here](https://stackoverflow.com/questions/14526627/double-http-status-header-on-http-post-to-jersey)
+and [Here](https://stackoverflow.com/questions/2964687/how-to-handle-100-continue-http-message).
+
+___
+### [OAuth](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc)
+##### Question:
+> Once I have a Authorization token, how do I use it to authenticate myself?
```
Comment (psvanstrom) [inline faq.adoc]: "Authorization token" should rather be named "Access token" to adhere to the OAuth specification.

**Comment by MrShish** [comment]
great initiative @ericsjogren !

**Comment by MrShish** [comment]
had to modify the permissions, commerce backend has write access now

File: faq.adoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<a href="url"><img src="https://d15n4q3o4x3svq.cloudfront.net/assets/tutorials/curl/api-a397cc184c5622fb5130af1b7baf149d.png" height="150" width="150" ></a>
+# Frequently Asked Questions
+This document contains answers to the most asked questions, if you have questions that are not answered
+in this document please refer to using our api-team email address: api@izettle.com .
+If you want to apply to receive access to our APIs, please fill in the form [here](https://docs.google.com/forms/d/e/1FAIpQLScdU9DDPfd680vhyZYAZ6DMIydQ1ikAOwP4abF2NWra5A-j8w/viewform)
```
Comment (psvanstrom) [inline faq.adoc]: The form URL that should be used is this one: https://www.izettle.com/api-access/

**Comment by guillermo-carrasco** [comment]
Awesome stuff, great! :dancer:

**Comment by alexanderzinolidis** [comment]
Great work! Are there any plans to create a developer documentation site of some kind with code examples? (Like Stripe, etc.)

**Comment by guillermo-carrasco** [comment]
@alexanderzinolidis there are plans to build a developer portal indeed. Unfortunately, no ETA... many projects going on right now!

**Pull request merged**